pkg/reconciler/v1alpha1/pipelinerun/pipelinerun.go

```diff
@@ -101,7 +101,7 @@ func NewController(
 		resourceLister:       resourceInformer.Lister(),
 	}
 
-	impl := controller.NewImpl(r, r.Logger, pipelineRunControllerName)
+	impl := controller.NewImpl(r, r.Logger, pipelineRunControllerName, reconciler.MustNewStatsReporter(pipelineRunControllerName, r.Logger))
```

Comment:
The pipelineRun reconciler also doesn't need this

Reply:
OK